evap/evaluation/management/commands/send_reminders.py

```diff
@@ -29,7 +29,7 @@ def handle(self, *args, **options):
 
         for recipient in recipients:
             due_courses = dict()
-            for course in Course.objects.filter(participants=recipient, state='in_evaluation').exclude(voters=recipient):
+            for course in Course.objects.filter(participants=recipient, state='in_evaluation', vote_end_date__in=check_dates).exclude(voters=recipient):
```

Comment:
yes, the reminder email is triggered if one course fulfills the condition but must list all courses that are still due.